src/topics/command-line-tool.md

```diff
@@ -10,6 +10,9 @@ The following example contains a minimal example of a CWL
 command-line tool for the `echo` Linux command, using inputs and
 outputs.
 
+How Command-lines work
```

Comment:
This needs to be capitalized like a tile. I would also suggest something else since this doesn't really describe how they work

Reply:
Hello @swzCuroverse thank you for the feedback, i have made a commit with changes.